src/Serilog/Serilog.csproj

```diff
@@ -23,25 +23,44 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'net45' ">
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
+    <PackageReference Include="System.ValueTuple" Version="4.3.1" />
```

Comment:
It's a pity. Why you didn't considered the fourth option (along with the first)? 
> 4.Support it everywhere, using reflection only (no direct dependency on  ValueTuple  itself)

Is that because of some performance concerns? I believe you could check `type.Name` to constants (`ValueTyple'1`, `ValueTuple'2` etc) like you did with `definition ==` comparisons